sp800-63b/sec5_authenticators.md

```diff
@@ -113,19 +113,19 @@ The purpose of the secret is to securely bind the authentication operation on th
 
 #### <a name="ooba"></a> 5.1.3.1 Out-of-Band Authenticators
 
-The out-of-band authenticator SHALL establish a separate channel with the verifier in order to retrieve the out-of-band secret or authentication request. This channel is considered to be out-of-band with respect to the primary communication channel, even if it terminates on the same device, provided the device does not leak information from one to the other without the authorization of the claimant.
+The out-of-band authenticator SHALL establish a separate channel with the verifier in order to retrieve the out-of-band secret or authentication request. This channel is considered to be out-of-band with respect to the primary communication channel - even if it terminates on the same device - provided the device does not leak information from one channel to the other without the authorization of the claimant.
 
-The out-of-band device SHOULD be uniquely addressable and communication over the secondary channel SHALL be encrypted unless sent via the public switched telephone network (PSTN). Methods that do not prove possession of a specific device, such as voice-over-IP (VOIP) or email, SHALL NOT be used for out-of-band authentication. For additional authenticator requirements specific to the PSTN, see [Section 5.1.3.3](#pstnOOB).
+The out-of-band device SHOULD be uniquely addressable and communication over the secondary channel SHALL be encrypted unless sent via the public switched telephone network (PSTN). For additional authenticator requirements specific to the PSTN, see [Section 5.1.3.3](#pstnOOB). Methods that do not prove possession of a specific device, such as voice-over-IP (VOIP) or email, SHALL NOT be used for out-of-band authentication.
 
-The out-of-band authenticator SHALL uniquely authenticate itself in one of the following ways in communicating with the verifier:
+The out-of-band authenticator SHALL uniquely authenticate itself in one of the following ways when communicating with the verifier:
 
-- Establish an authenticated protected channel to the verifier using approved cryptography. The key used SHALL be stored in suitably secure storage available to the authenticator application (e.g., keychain storage, trusted platform module, secure element, trusted execution environment).
+- Establish an authenticated protected channel to the verifier using approved cryptography. The key used SHALL be stored in suitably secure storage available to the authenticator application (e.g., keychain storage, trusted platform module [TPM], secure element, trusted execution environment [TEE]).
```

Comment:
This is the first appearance of the acronym and typically that would be put in parens. But all of them are listed within the parenthesis so you would use brackets, correct?